internal/hardware/validator.go

```diff
@@ -411,8 +411,12 @@ func (v *validator) getOCPRequirementsForVersion(openshiftVersion string) (*mode
 	return v.VersionedRequirements.GetVersionedHostRequirements(openshiftVersion)
 }
 
-func (v *validator) getValidDeviceStorageTypes() []string {
-	return []string{string(models.DriveTypeHDD), string(models.DriveTypeSSD), string(models.DriveTypeMultipath)}
+func (v *validator) getValidDeviceStorageTypes(hostArchitecture string) []string {
+	validTypes := []string{string(models.DriveTypeHDD), string(models.DriveTypeSSD), string(models.DriveTypeMultipath)}
+	if hostArchitecture == models.ClusterCPUArchitectureS390x {
```

Comment:
do we need to also check that it's not null?

Reply:
hostArchitecture is a string, it can't be null